app/database/migrations/2015_10_01_024551_add_accessory_consumable_price_info.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddAccessoryConsumablePriceInfo extends Migration {
+
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		Schema::table('accessories', function ($table) {
+			$table->date('purchase_date')->nullable();
```

Comment:
FYI, we always need to use `->default(NULL)` on these, or stuff will blow up in strict mode.